openstack/orchestration/v1/stacks/doc.go

```diff
@@ -5,4 +5,121 @@
 // application framework or component specified (in the template). A stack is a
 // running instance of a template. The result of creating a stack is a deployment
 // of the application framework or component.
+
```

Comment:
More documentation is always better than no documentation, so we can definitely keep everything here. However, let's strip out all `//` and `/* */` marks from this document and just have one `/*` at the beginning and one `*/` at the end of this file.